### PR TITLE
Removed `side petros` #282 again but hopefully no conflicts

### LINKED ISSUE #282: Avoid 'side petros' Checks
The gamemode has a real dependency on Petros being alive, but situations keep cropping up where he's dead, even temporarily. 

The main issue it's causing is players that join can't access the arsenal, or get the wrong loadout, or the wrong spawn, as well as other things.

To start - remove 'side petros' from all the init code (initPlayerLocal and initServer)

### PR DESCRIPTION
**Pull request by tpaclatee**
## What type of PR is this.
1. [x] Bug
2. [ ] Change
3. [ ] Enhancement

### What have you changed and why?
Information:
    Attempted to change `side petros` and `side (group petros)` in all checks for player side. Bug was caused by dead petros on check.

### Please specify which Issue this PR Resolves.
closes #282 

### Please verify the following and ensure all checks are completed.

2. [x] Have you loaded the mission in LAN host?
3. [x] Have you loaded the mission on a dedicated server? *by @CalebSerafin*

### Is further testing or are further changes required?
1. [ ] No
2. [x] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 
Not tested, needs further testing
********************************************************
Notes:
Untested change, found six instances where `side petros` was checked and removed

### CODE REVIEW

File: A3-Antistasi/functions/init/fn_initServer.sqf [hunk 1]
```diff
@@ -88,7 +88,7 @@ call
 
 	// Otherwise, check through the saved game list for matches and build existing ID list
 	private _saveList = [profileNamespace getVariable "antistasiSavedGames"] param [0, [], [[]]];
-	private _gametype = if (side petros == independent) then {"Greenfor"} else {"Blufor"};
+	private _gametype = if (teamPlayer == independent) then {"Greenfor"} else {"Blufor"};
```
Comment (CalebSerafin) [inline A3-Antistasi/functions/init/fn_initServer.sqf:91]: InitVar is only called on Ln 120, so this one should also be left as `side Petros`
https://github.com/official-antistasi-community/A3-Antistasi/blob/406cf380369be48330a9a7f621cf24771a4bf1c2/A3-Antistasi/functions/init/fn_initServer.sqf#L120

Comment (CalebSerafin) [reply A3-Antistasi/functions/init/fn_initServer.sqf:91]: Or another possibility is to move `call A3A_fnc_initVar; ` earlier. Testing will be required to determine if there are any consequences.

Comment (tpaclatee) [reply A3-Antistasi/functions/init/fn_initServer.sqf]: changed back to `side petros` from `teamPlayer`

File: A3-Antistasi/functions/Ammunition/fn_itemSort.sqf [hunk 2]
```diff
@@ -1,10 +1,10 @@
-if (side (group petros) == west) then {
+if (teamPlayer == west) then {
```
Comment (CalebSerafin) [inline A3-Antistasi/functions/Ammunition/fn_itemSort.sqf:1]:
```suggestion
if (teamPlayer isEqualTo west) then {
```

File: A3-Antistasi/functions/Ammunition/fn_itemSort.sqf [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
 	diveGear append ["U_B_Wetsuit","V_RebreatherB","G_Diving"];
 } else {
 	diveGear append ["U_I_Wetsuit","V_RebreatherIA","G_Diving"];
 };
 
-if (side (group petros) == west) then {
+if (teamPlayer == west) then {
```
Comment (CalebSerafin) [inline A3-Antistasi/functions/Ammunition/fn_itemSort.sqf:7]:
```suggestion
if (teamPlayer isEqualTo west) then {
```

File: A3-Antistasi/functions/Save/fn_saveLoop.sqf [hunk 4]
```diff
@@ -23,7 +23,7 @@ private _saveIndex = -1;
 
 // If not, append a new entry
 if (_saveIndex == -1) then {
-	private _gametype = if (side petros == independent) then {"Greenfor"} else {"Blufor"};
+	private _gametype = if (teamPlayer == independent) then {"Greenfor"} else {"Blufor"};
```
Comment (CalebSerafin) [inline A3-Antistasi/functions/Save/fn_saveLoop.sqf:26]:
```suggestion
	private _gametype = if (teamPlayer isEqualTo independent) then {"Greenfor"} else {"Blufor"};
```

File: A3-Antistasi/functions/Save/fn_varNameToSaveName.sqf [hunk 5]
```diff
@@ -4,9 +4,9 @@ params ["_varName", ""];
 if (worldName == "Tanoa") then {
 	format["%1%2%3%4",_varName,serverID,campaignID,"WotP"];
 } else {
-	if (side group petros == independent) then {
+	if (teamPlayer == independent) then {
```
Comment (CalebSerafin) [inline A3-Antistasi/functions/Save/fn_varNameToSaveName.sqf:7]:
```suggestion
	if (teamPlayer isEqualTo independent) then {
```

File: A3-Antistasi/functions/init/fn_initClient.sqf [hunk 6]
```diff
@@ -81,7 +81,7 @@ else {
 private ["_colourTeamPlayer", "_colorInvaders"];
 _colourTeamPlayer = teamPlayer call BIS_fnc_sideColor;
 _colorInvaders = Invaders call BIS_fnc_sideColor;
-_positionX = if (side player == side (group petros)) then {position petros} else {getMarkerPos "respawn_west"};
+_positionX = if (side player == teamPlayer) then {position petros} else {getMarkerPos "respawn_west"};
```
Comment (CalebSerafin) [inline A3-Antistasi/functions/init/fn_initClient.sqf:84]:
```suggestion
_positionX = if (side player isEqualTo teamPlayer) then {position petros} else {getMarkerPos "respawn_west"};
```

**Comment by CalebSerafin** [comment]
Tested on DS, no issues found so far.
[arma3server_x64_2020-07-12_13-06-10.rpt.txt](https://github.com/official-antistasi-community/A3-Antistasi/files/4908947/arma3server_x64_2020-07-12_13-06-10.rpt.txt)

**Pull request merged**